src/test/java/com/cloudant/tests/HttpProxyTest.java

```diff
@@ -63,12 +63,10 @@ public void proxyConfiguration() throws Exception {
         for (String line : server.getLastInputRequestLines()) {
             if (line.contains("Proxy-Authorization")) {
                 foundProxyAuthHeader = true;
-                Matcher m = Pattern.compile("Proxy-Authorization\\s*:\\s*(.*)", Pattern
+                Matcher m = Pattern.compile("Proxy-Authorization: Basic (.*)", Pattern
                         .CASE_INSENSITIVE).matcher(line);
                 assertTrue("The Proxy-Authorization header should match the pattern", m.matches());
-                assertEquals("There should be 2 groups, one for the header key and one for the " +
-                        "value", 2, m.groupCount());
-
+                assertEquals("There should be 1 group for the value", 1, m.groupCount());
```

Comment:
Why did the group count change? I thought you got one group for the whole matching string and another for the bracketed subgroup, hence the original 2 - is that wrong?